target/linux/ramips/image/mt76x8.mk

```diff
@@ -19,6 +19,16 @@ endef
 DEVICE_VARS += TPLINK_FLASHLAYOUT TPLINK_HWID TPLINK_HWREV TPLINK_HWREVADD TPLINK_HVERSION
 
 
+define Device/axing_w06
+  DTS := W06
+  IMAGE_SIZE := 15040k
+  DEVICE_TITLE := Axing W06
+  DEVICE_PACKAGES := kmod-usb2 kmod-usb-ohci kmod-ledtrig-netdev
+  IMAGES += factory.bin
+  IMAGE/factory.bin := $$(sysupgrade_bin) | check-size $$$$(IMAGE_SIZE)
```

Comment:
Am I right that the stock firmware doesn't like our sysupgrade image with metadata appended?

Reply:
Yes, stock firmware for W06 is based on OpenWrt, but it does'nt have metadata.

Reply:
If the stock firmware is OpenWrt our mtadata appended sysupgrade image should work out of the boy. please test an upgrade from stock firmware to self build openwrt using the sysupgrade.bin image!

Reply:
I see...
Using Sysupgrade.bin, I was able to rewrite from stock firmware to openwrt without any problems.
I'll drop the factory.bin image.